src/common/utils/helpers.js

```diff
@@ -29,5 +29,24 @@ export const getFiatFromSatoshi = async (currency, satoshi) => {
 };
 
 export const calcFiatFromSatoshi = (exchangeRate, satoshi) => {
```

Comment:
wondering if we should use obeject parameters here which might be more descriptive?

`calcFiatFromSatoshi({rate: xxx, satoshi: xxx})`

Reply:
I'd do this on more parameters but not on two - up to you

Reply:
yeah, don't mind now. just a thought, generally I prefer that better as it makes it easier for people to read.